source/get-started/patternfly-migration-guides/angular-patternfly-migration-guide-3.0.md

```diff
@@ -0,0 +1,84 @@
+---
+title: PatternFly Angular 3.0 Migration Guide
+author: dtaylor / ajolicoeur
+layout: page
+---
+## Angular-PatternFly 3.0 Migration Guide
+
+This guide details how to upgrade from PatternFly Angular 2.x to our latest version, [PatternFly Angular 3.0][1]. Using base PatternFly? Use the [PatternFly 3.0][8] upgrade instead.
+
+### Release Notes:
+
+  * Updated devDependency to PatternFly for 3.0.0
```

Comment:
I think your right.  Typically, migration guides document a migration to a major release, so let's just keep it at 3.0.0